src/components/forms/__tests__/PhoneInput.test.tsx

```diff
@@ -0,0 +1,137 @@
+import * as React from "react"
+import { screen, render } from "@testing-library/react"
+import user from "@testing-library/user-event"
+
+import { PhoneInput, CountryCode } from "../PhoneInput"
+
+describe("PhoneInput", () => {
+  const onChange = jest.fn()
+
+  afterEach(() => onChange.mockClear())
+
+  describe("props", () => {
+    describe("initialCountryCode", () => {
+      describe("AU", () => {
+        beforeEach(() => {
+          render(
+            <PhoneInput
+              onChange={onChange}
+              initialCountryCode={CountryCode.AU}
+            />
+          )
+        })
+
+        it("shows Australia's calling code", () => {
+          expect(screen.getByText("+61")).toBeInTheDocument()
+        })
+
+        it("has Australia selected in dropdown", () => {
+          const selectComponent = screen.getByRole("option", {
```

Comment:
`selectComponent` isn't technically an accurate name here - perhaps we can rename it to `selectedOption` or simply `option`